src/devices/yokis.ts

```diff
@@ -1400,6 +1400,37 @@ const yokisCommandsExtend = {
     },
 };
 
+// Yokis specific definition
+
+// Yokis does not support the timer OnOff cluster. It uses a custom cluster instead.
+// Dereferencing the `on_time` and `off_wait_time` from the keys of the converter.
+const yokisTz = {
+    on_off: {
+        key: ['state'],
```

Comment:
```suggestion
        key: ['state'],
        ...tz.on_off,
```

the rest of the code can be removed.

Reply:
Updated : i had to put the spread operator first as it was overriding the key.